_posts/2020-08-23-gsoc-20-tf-2-examples.md

```diff
@@ -0,0 +1,62 @@
+
+# Aim
+
+[Kubernetes](https://kubernetes.io/) is already an industry-standard in managing cloud resources. [Kubeflow](https://www.kubeflow.org/) is on its path to become an industry standard in managing machine learning workflows on cloud. Examples that illustrate Kubeflow functionalities using latest industry technologies make Kubeflow easier to use and more accessible to all potential users. This project has aimed at building samples for Jupyter notebook to Kubeflow deployment using Tensorflow 2.0 Keras for backend training code, illustrating customer user journey (CUJ) in the process. This project has also served as an hands-on to large scale application of machine learning bringing in the elements of DevOps and SRE and this has kept me motivated throughout the project.
+
+# The Kubeflow Community
+
+The [Kubeflow community](https://www.kubeflow.org/docs/about/community/) is a highly approachable and closely-knit community that has been reaching out to and [helping](https://www.kubeflow.org/docs/about/gsoc/) potential GSoC students well before the application period. Respecting this, I made sure to take feedback for my proposal of the [project idea](https://summerofcode.withgoogle.com/projects/#5507335985823744) I chose, before the application deadline. Mentors [Yuan Tang](https://github.com/terrytangyuan), [Ce Gao](https://github.com/gaocegege) and [Jack Lin](https://github.com/ChanYiLin) were candid in providing me feedback and I refined and changed my proposal accordingly. To my sweet surprise, I got selected for the idea!😁 What has really helped me in these three months of coding period is that one month of [community bonding](https://developers.google.com/open-source/gsoc/timeline) where I got to know the community and more about the technicalities of Kubeflow.
+
+# The Project
+
+Examples created as part of this project needed to be easily reproducible to serve their purpose. Initially the underlying model decided to demonstrate Kubeflow functionalities was a BiDirectional RNN to be trained on IMDB large movie review [dataset](http://ai.stanford.edu/%7Eamaas/data/sentiment/) for sentiment analysis based on a [tensorflow tutorial](https://www.tensorflow.org/tutorials/text/text_classification_rnn). Over the course of time, we decided to also add another set of examples using  a neural machine translation model in its backend trained on a Spanish to English [dataset](http://www.manythings.org/anki/) based on another [tensorflow tutorial](https://www.tensorflow.org/tutorials/text/nmt_with_attention).
+
+The reasons for choosing these models were -
+* The [kubeflow/examples](https://github.com/kubeflow/examples) repo needed more NLP-related tasks.
```

Comment:
Perhaps some images/graphics in the blog would be useful to make the blog/project look more interesting

Reply:
Added